spec.txt

```diff
@@ -1739,8 +1730,8 @@ baz</p>
 An [indented code block](@) is composed of one or more
 [indented chunks] separated by blank lines.
 An [indented chunk](@) is a sequence of non-blank lines,
-each indented four or more spaces. The contents of the code block are
-the literal contents of the lines, including trailing
+each indented four or more spaces of indentation. The contents of the code block
```

Comment:
"indented four or more spaces of indentation" seems a bit redundant.
Remove "of indentation"?

Reply:
I'm going to move this discussion to the main thread, since it gets lost here.